src/components/onboarding/sign-up/SignUp.js

```diff
@@ -0,0 +1,256 @@
+import React, { Component } from 'react'
+import * as ReactRedux from 'react-redux'
+import PropTypes from 'prop-types'
+import { compose } from 'recompose'
+import { withStyles } from '@material-ui/core/styles'
+
+import authActions from 'redux/actions/auth'
+
+const styles = {
```

Comment:
I would suggest putting CSS in the separate file called *.style.js

Reply:
You're right, it should be *.style.js

Reply:
file deleted